package.json

```diff
@@ -109,7 +109,7 @@
         "kea-window-values": "^3.0.0",
         "md5": "^2.3.0",
         "monaco-editor": "^0.23.0",
-        "posthog-js": "1.50.3",
+        "posthog-js": "1.50.2",
```

Comment:
you'll also need to update the lockfile. Or just revert the 1.50.3 PR which will do both automatically

Reply:
ha yeah sorry haven't had my coffee yet, just updated the lock file